src/index.ts

```diff
@@ -242,10 +243,45 @@ async function wranglerCommands() {
 				}
 			}
 
-			await exec(`${packageManager.exec} wrangler ${command}`, args, {
+			// Used for saving the wrangler output
+			let stdOut = "";
+			let stdErr = "";
+
+			// Construct the options for the exec command
+			const options = {
 				cwd: config["workingDirectory"],
 				silent: config["QUIET_MODE"],
-			});
+				listeners: {
+					stdout: (data: Buffer) => {
+						stdOut += data.toString();
+					},
+					stderr: (data: Buffer) => {
+						stdErr += data.toString();
+					},
+				},
+			};
+
+			// Execute the wrangler command
+			await exec(`${packageManager.exec} wrangler ${command}`, args, options);
+
+			// Set the outputs for the command
+			setOutput("command-output", stdOut);
+			setOutput("command-stderr", stdErr);
```

Comment:
the `command` input accepts multiline strings and will execute each line as a separate command execution. as currently implemented, `command-output` and `command-stderr` will only capture output from the last executed command within the loop. this is _probably ok_ for now, but I wonder if it would be more useful to collect (concatenated) output from all command executions, or if that would be confusing without some delimiter between execution results.

Reply:
I am personally and end-user of the exact changes that I am implementing and I would personally find the extra output of other commands in these variables to be confusing. If we added concatenated output in some form, (via a delimiter) this would likely lead to even more string formatting / processes that I was trying to ultimately get farther away from with these changes. For now, I think this is okay and would be willing to iterate on a future pull request if folks think that would be a good idea or have different use cases in the future.

Reply:
I’m ok with that.